app/views/payments/index.html.slim

```diff
@@ -19,6 +20,7 @@
             td= number_to_currency(payment.amount)
             td= payment.date
             td= payment&.category&.name&.capitalize
+            td= payment.notes
```

Comment:
I am fine with this for now, but in future I'll rewrite method show for payments. And it will show modal window with the picture, and other necessary information.